qualtran/surface_code/ccz2t_cost_model.py

```diff
@@ -150,17 +149,20 @@ def n_cycles(self, n_magic: AlgorithmSummary) -> int:
 
         return math.ceil((n_ccz_states * ccz_depth + catalyzations) * distillation_d)
 
+    def spacetime_footprint(self) -> float:
+        raise NotImplementedError
```

Comment:
()

Reply:
actually... why is this not implemented?

Reply:
done